miso-web/src/main/webapp/scripts/form_sample.js

```diff
@@ -396,8 +396,8 @@ FormTarget.sample = (function($) {
               }
             }]
           }, {
-            title: 'LCM Tube',
-            include: config.detailedSample && object.sampleSubcategory === 'LCM Tube',
+            title: 'Tissue Pieces',
+            include: config.detailedSample && object.sampleSubcategory === 'Tissue Piece',
             fields: [{
               title: 'Slides Consumed',
```

Comment:
Only display if parent class is Slides? Same in HOT

Reply:
Never mind. Doesn't make sense since you could edit samples with different parent classes at the same time in HOT